### PR TITLE
vassal: new package

### PR DESCRIPTION
**Pull request by tvestelind**
I have created a new package for VASSAL, an open-source boardgame engine.

I have tested it manually by building, launching and playing around a bit. I loaded one game module and it worked.

One thing that I noticed that didn't work was the help, but I think it's because VASSAL cannot find a browser, because I have made sure that the path to the help file is correct.

### CODE REVIEW

**Comment by ktosiek** [comment]
You haven't added yourself to maintainers: https://travis-ci.org/NixOS/nixpkgs/builds/104306248#L157
Have you missed a file when creating the commit?

**Comment by tvestelind** [comment]
@ktosiek thanks. I added myself when adding another package so I forgot to do it in this as well :)

File: pkgs/games/vassal/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    makeWrapper ${jre}/bin/java $out/bin/vassal \
+      --add-flags "-Duser.dir=$out -cp $out/share/vassal/Vengine.jar \
+      VASSAL.launch.ModuleManager"
+  '';
+
+  # Don't move doc to share/, VASSAL expects it to be in the root
+  forceShare = [ "man" "info" ];
+
+  meta = with stdenv.lib; {
+      description = "A free, open-source boardgame engine.";
```
Comment (pSub) [inline pkgs/games/vassal/default.nix]: Descriptions should not end with a period.

Comment (tvestelind) [inline pkgs/games/vassal/default.nix]: Fixed.

**Comment by lethalman** [comment]
Travis passed, thanks.

**Pull request merged**